### PR TITLE
Cell and synapse labels

### PR DESCRIPTION
**Pull request by ceesem**
A first draft of tags for labeling of cell types by valence, IVSCC type, and classical neuroanatomy names using bound spatial points.

### CODE REVIEW

File: emannotationschemas/config.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import json
+
+def import_config( schema_config_file ):
```
[the remark this answers is not in the document]
Comment (ceesem) [reply emannotationschemas/config.py]: Included by mistake, yup.

File: emannotationschemas/postsynaptic_compartment.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+from emannotationschemas.base import ReferenceAnnotation
+import marshmallow as mm
+from marshmallow.validate import OneOf
+
+allowed_compartments = ['soma',
```
[the remark this answers is not in the document]
Comment (ceesem) [reply emannotationschemas/postsynaptic_compartment.py]: Agreed, more splits are much better here.

Current first thought:
* compartment : OneOf( ['soma','dendrite','axon'] ), required
* on_spine : Boolean, optional
* dendrite_type : OneOf( ['basal', 'apical'] ), optional

File: emannotationschemas/__init__.py [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
+from emannotationschemas.cell_type_local import CellTypeLocal
 __version__ = '0.2.0'
 
 type_mapping = {
     'synapse': SynapseSchema,
-    'bouton_shape': BoutonShape,
-    'functional_coregistration': FunctionalCoregistration
+    'presynaptic_bouton_type': PresynapticBoutonType,
+    'postsynaptic_compartment': PostsynapticCompartment,
+    'cell_type': CellTypeLocal,
```
Comment (sdorkenw) [inline emannotationschemas/__init__.py]: use more specific name

File: emannotationschemas/postsynaptic_compartment.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    compartment = mm.fields.Str(
+        required=True,
+        validate=OneOf(allowed_compartments),
+        description='Compartment of the postsynaptic neuron \
+                      targeted by the synapse'
+        )
+
+    on_spine = mm.fields.Bool(
+        required=False,
+        description='Boolean representing if the synapse is onto a spine or not')
```
Comment (sdorkenw) [inline emannotationschemas/postsynaptic_compartment.py]: This ignores the spine neck

File: emannotationschemas/cell_type_local.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+                             'martinotti',
+                             'pv',
+                             'sst',
+                             'vip',
+                             'neurogliaform',
+                             'unknown',
+                             'astrocyte',
+                             'microglia-perivascular',
+                             'microglia-perineuronal',
+                             'unknown'
```
Comment (sdorkenw) [inline emannotationschemas/cell_type_local.py]: 'unknown' is duplicated

**Pull request merged**